ehrapy/tools/supervised/_feature_importances.py

```diff
@@ -0,0 +1,157 @@
+from typing import Literal
+
+import pandas as pd
+from anndata import AnnData
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler, StandardScaler
+from sklearn.svm import SVC, SVR
+
+from ehrapy import logging as logg
+
+
+def feature_importances(
+    adata: AnnData,
+    predicted_feature: str,
+    prediction_type: Literal["continuous", "categorical"],
+    model: Literal["regression", "svm", "rf"] = "regression",
+    input_features: list[str] | Literal["all"] = "all",
```

Comment:
This could probably also be of type `Iterable` instead of `list`. We try to always use the most generic type possible to not constrain users. "possible" here means: Without introducing too much complexity and still KISS it